docs/src/Box.doc.js

```diff
@@ -225,390 +247,487 @@ card(
-    defaultCode={`
+Box is a pass-through component, meaning that any other properties you provide to it will be directly applied to the underlying \`<div>\`. The above properties are exceptions, however.  We don’t allow  \`onClick\`  for  accessibility reasons, so consider a [Button](/Button) or [TapArea](/TapArea) instead. We remove \`className\` and \`style\` to ensure style encapsulation. If necessary, \`dangerouslySetInlineStyle\` can be used to supply a style not supported by Box props.
+
+If you need to use these features for animation purposes, use a \`<div>\` instead.`}
+        defaultCode={`
 <Box
-  alignItems="center"
-  direction="row"
-  display="flex"
-  marginStart={-1}
-  marginEnd={-1}
+  className="This class name will not appear"
+  style={{backgroundColor: "orange"}}
+  color="midnight"
+  column={12}
+  height={100}
+  padding={4}
 >
-  <Box paddingX={1}>
-    <Avatar name="chrislloyd" size="md" />
+  <Box
+    onClick={() => {console.log("This won't get logged.");}}
+    paddingX={1}
+    color="maroon"
+    height={50}
+  >
+    <Text color="white" weight="bold">Clicking here will do nothing</Text>
   </Box>
-  <Box paddingX={1}  flex="grow">
-    <Text weight="bold">Chris Lloyd</Text>
-    <Text>joined 2 years ago</Text>
+</Box>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description={`When addressing the spacing of the Box, use padding before you use margins, as padding will compose better and won't collapse. Padding is applied in 4px increments and is always symmetric. Learn more about [margin collapsing]("https://css-tricks.com/what-you-should-know-about-collapsing-margins/"). `}
+        defaultCode={`
+<Box>
+  <Box marginTop={4} color="blue" width={200} height={50}>
+    <Box
+      color="green"
+      height={50}
+      marginTop={2}
+    >
+      <Box
+        color="midnight"
+        height={50}
+        marginTop={3}
+      >
+        <Text color="white" weight="bold">
+          These all collapsed
+        </Text>
+      </Box>
+    </Box>
   </Box>
-  <Box paddingX={1}>
-    <Button text="Follow" size="sm" color="red" />
+  <Box marginTop={4} paddingY={1} color="blue" width={200} height={100}>
+    <Box
+      color="green"
+      height={100}
+      paddingY={2}
+    >
+      <Box
+        color="midnight"
+        height={100}
+        padding={2}
+      >
+        <Text color="white" weight="bold">
+          These are not collapsed, because they use padding
+        </Text>
+      </Box>
+    </Box>
   </Box>
 </Box>
 `}
-  />,
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description={`Avoid using arbitrary \`<div>\` elements. Instead, when building a component, prioritize using Box. If you need to set a custom style, you can do so using the \`dangerouslySetInlineStyle\` prop. However, this should be avoided whenever possible by utilizing the other props provided in Box.`}
```

Comment:
You may also want to mention / link to our lint rule for this

Reply:
good call